middle_end/flambda2/tests/mlexamples/closure_var_use_b.ml

```diff
@@ -0,0 +1,9 @@
+external op : 'a -> 'a = "%opaque"
+
+module A = Closure_var_use_a
+
+let _ =
+  let g =
+    if op true then (A.f [@inlined always]) 0 else (A.f [@inlined always]) 1
+  in
+  (g [@inlined never]) 2
```

Comment:
It would be good to have a brief comment here explaining what state we've gotten `g` into.